cmd/tfengine/README.md

```diff
@@ -6,8 +6,9 @@ The Terraform Engine is a tool to generate complete end-to-end Terraform
 deployments for Google Cloud with security, compliance, and best practices baked
 in.
 
-It introduces the concept of "templates" to wrap Terraform modules. Templates
-define configuration specific to your Google Cloud organization and structure.
+It introduces the concept of "templates". Templates can be used to manage
```

Comment:
I don't think it's quite accurate to say templates "manage" TF modules. You use them to generate modules. I suggest rewording "manage".